### ISSUE TITLE
GGMAX - Build 2022.05.13 - Particle Issue

### ISSUE THREAD

**Issue by Necrym59** [TheGameCreators/GameGuruRepo#3090]
The option to show at start of level if unticked does not seem to have a way of activating that hidden particle nor an activating command in lua that i can see.
![particle](https://user-images.githubusercontent.com/95015657/167732743-89d968a7-0595-4b38-8a4f-9d0b9268b1eb.png)
.

**Comment by Necrym59** [comment]
It works ONLY with a logic connection with a triggerzone nothing else. Not with any other object using the ActivateIfUsed(e) function does nor does it activate if you use Hide(e) on a particle you cannot then activate it it should be able to be hidden and activated easily its needed for a rocket trail for example or sudden fire or smoke effects troggered by scripts or other objects.

**Comment by LeeBamberTGC** [comment]
NOTE: For now we are depreciating the old IFUSED string that is now quite buried in the developer mode properties of an object, and relying on the Visual Logic system instead to make the activation/deactivation of items more intuitive for more users.

I have set this as a feature request so we can continue the conversation on what specific types of things we want behaviors to do that cannot be done currently with the Visual Logic system.

**Comment by Necrym59** [comment]
The only problem with the Visual Logic system is the 'distance' of connections. Its great for smaller local areas but over a long distance is extremly awkward, maybe a click once (not hold) then (click once) on the distant object maybe a solution for long connections. Also the longer the connection it gets smaller to see and the option ball in the centre of the long connection is not easy to find.  The IfUsed field comes in handy for such instances so should not be completly unavailable but in the Dev section is ok for now.

**Comment by LeeBamberTGC** [comment]
@Necrym59 Yes I think the CLICK move CLICK method might work out better for a number of reasons. It would solve the distance issue, and also some users are not comfortable clicking and holding and dragging and moving and getting the right destination, etc.  It also means we can prompt some instructions between the clicks.
